docs/rst/manual/reference/configuration/magic_strings.rst

```diff
@@ -0,0 +1,259 @@
+will typically be replaced with different values depending on the current iteration or
+realization index.
+
+**It should also be noted that experiment variables can only be used with forward model jobs
+and not with workflows.**
+
+
+List of keywords
+----------------
+
+
+* Experiment Constants
+
+   * :ref:`RUNPATH_FILE <ms_runpath_file>`
+   * :ref:`CONFIG_PATH <ms_config_path>`
+   * :ref:`CONFIG_FILE <ms_config_file>`
+   * :ref:`CONFIG_FILE_BASE <ms_config_file_base>`
+   * :ref:`CWD <ms_cwd>`
+
+* Experiment Variables
+
+   * :ref:`IENS <ms_iens>`
+   * :ref:`ITER <ms_iter>`
+   * :ref:`ECLBASE/ECL_BASE <ms_eclbase>`
+   * :ref:`GEOID <ms_geoid>`
+   * :ref:`RUNPATH <ms_runpath>`
+   * :ref:`NUM_CPU <ms_num_cpu>`
+
+
+
+.. _ms_exp_constants:
+
+Experiment Constants
+--------------------
+
+.. _ms_runpath_file:
+.. topic:: RUNPATH_FILE
+
+    Will be replaced with an absolute path to the :ref:`RUNPATH_FILE <runpath_file>`.
+
+   **Example:**
+
+   ::
+
+      YOUR_WORKFLOW_JOB <RUNPATH_FILE>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB /path/to/the/runpath/file/.ert_runpath_list
+
+.. _ms_config_path:
+.. topic:: CONFIG_PATH
+
+   Will be replaced with the path to the directory containing the config file for the experiment.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_PATH>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB /path/to/config/
+
+.. _ms_config_file:
+.. topic:: CONFIG_FILE
+
+   Will be replaced with the name of the config file for the experiment.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_FILE>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB example.ert
+
+.. _ms_config_file_base:
+.. topic:: CONFIG_FILE_BASE
+
+   Will be replaced with the base name of the config file for the experiment.
+   This is the filename without the extension.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CONFIG_FILE_BASE>
+
+   Given that the filename is `example.ert`, this will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB example
+
+.. _ms_cwd:
+.. topic:: CWD
+
+   Will be replaced with the current working directory.
+   This will usually be the same as the :ref:`CONFIG_PATH <ms_config_path>`.
+
+
+   *Example:*
+
+   ::
+
+      YOUR_WORKFLOW_JOB <CWD>
+
+   Will be replaced with the following at runtime:
+
+   ::
+
+      YOUR_WORKFLOW_JOB /data/repos/ert/test-data/local/poly_example
+
+.. _ms_exp_variables:
+
+Experiment Variables
+--------------------
+
+.. warning::
+   The experiment variables cannot be used with workflows, only forward model jobs!
+
+.. _ms_iens:
+.. topic:: IENS
+
+   Will be replaced with the index for the current realization.
+   The realization number uses zero-based indexing.
+
+
+   *Example:*
+
+   ::
+
+      SIMULATION_JOB some_forward_model_job <IENS>
```

Comment:
Should we use the `FORWARD_MODEL` keyword for the example instead? I'm suspecting `SIMULATION_JOB` will never be promoted for the users? 🤷